test/events.test.js

```diff
@@ -18,7 +18,7 @@ test('no event loop logs successfully', function (t) {
   }))
 
   child.on('close', function () {
-    t.notEqual(output.match(/"msg":"h"/), null)
+    t.notEqual(null, output.match(/"msg":"h"/))
```

Comment:
If we are going to change all of these it might as well to be to using `t.match()`.